root/static/scripts/common/components/WorkListEntry.js

```diff
@@ -47,15 +47,19 @@ export const WorkListRow = ({
     ) : null}
     <td><EntityLink entity={work} /></td>
     <td>
-      <ArtistRoles relations={work.writers} />
+      {work.writers.length
+        ? <ArtistRoles relations={work.writers} />
+        : l('-')}
```

Comment:
The only bit where I'm not quite sure. This hyphen was only here for reports, it seems, but it wasn't there for searches and other work lists. IMO just an empty cell is a better choice here (to represent "nothing entered yet") than hyphen. If we're missing a type, gender, or whatnot for an artist we don't use a hyphen, we just leave it blank (https://musicbrainz.org/search?query=a&type=artist&limit=25&method=indexed) - I don't see why this should be different.

Reply:
@reosarevok:  I removed that commit from this pull request and made it available to you at https://github.com/yvanzo/musicbrainz-server/commit/ee3c531495daecb44f361049c79a2354d14fe0e3.